Demo/Demo/Views/CustomButton.swift

```diff
@@ -1,9 +1,15 @@
 import UIKit
 
 class CustomButton: UIButton {
+    var titleHolder: String?
+    // swiftlint:disable:next implicitly_unwrapped_optional
```

Comment:
Should we turn off the linter for our Demo app, only have it run on the `PayPal.xcodeproj`?

Reply:
I like the idea of running it on both just to keep style consistent across the SDK and demo. But I'll update this so that I don't need to force unwrap and can remove this linter comment. 😄